web/src/features/feature-flags/api.ts

```diff
@@ -0,0 +1,40 @@
+import { useQuery } from '@tanstack/react-query';
+import { QUERY_KEYS, getBasePath, getHeaders } from 'api/helpers';
+import { FeatureFlags } from './types';
+
+export async function getFeatureFlags(): Promise<FeatureFlags> {
+  const path = '/feature-flags';
+  const requestOptions: RequestInit = {
+    method: 'GET',
+    headers: await getHeaders(path),
+  };
+
+  try {
+    const response = await fetch(`${getBasePath()}${path}`, requestOptions);
+    if (response.ok) {
+      const data = await response.json();
+      return data;
```

Comment:
Same here.
```suggestion
      return await response.json();
```

Reply:
I personally think it's easier to read and understand the more explicit approach (as "return await" might require a bit of extra mental processing), but I'm also fine with changing it :)